requirements.txt

```diff
@@ -1,19 +1,14 @@
-aiofiles==23.1.0
+aiofiles==23.2.1
+aiohttp==3.8.5
 aiohttp_cors==0.7.0
-aiohttp==3.8.4
-black==23.7.0
-coverage==7.2.7
 defusedxml==0.7.1
 frozendict==2.3.8
-globus-sdk==1.6.1
-hypothesis==6.81.1
+globus-sdk==3.28.0
```

Comment:
I don't think we can upgrade globus

Reply:
Ok, tests pass either way, I'll revert to the old version.